tasks/_apt-key-import.yml

```diff
@@ -18,71 +19,75 @@
   set_fact:
     key_needs_import: "{{ 'false' if key_exists_result.rc == 0 else 'true' }}"
   when: key_fingerprint != datadog_apt_key_current_name
 
-- name: "Set local helper variable for determining key import (when {{ datadog_apt_key_current_name }})"
+- name: Set local helper variable for determining key import (when {{ datadog_apt_key_current_name }})
   set_fact:
     key_needs_import: "true"
   when: key_fingerprint == datadog_apt_key_current_name
 
-- name: "Create temporary directory for key manipulation"
+- name: Create temporary directory for key manipulation
   tempfile:
     state: directory
     suffix: keys
   register: tempdir
   when: key_needs_import
   changed_when: false
 
-- name: "Download {{ keyring_url }} to import key {{ key_fingerprint }}"
+- name: Download url to import key. url,key = {{ keyring_url, key_fingerprint }}
   get_url:
     url: "{{ keyring_url }}"
     dest: "{{ tempdir.path }}/{{ key_fingerprint }}"
-    force: yes
+    force: true
+    mode: 600
   when: key_needs_import
   changed_when: false
 
 # gpg --dearmor called on a binary keyring does nothing
-- name: "Ensure downloaded file for {{ key_fingerprint }} is a binary keyring"
-  shell: "cat {{ tempdir.path }}/{{ key_fingerprint }} | gpg --dearmor > {{ tempdir.path }}/binary.gpg"  # noqa risky-shell-pipe
+- name: Ensure downloaded file for key is a binary keyring, key={{ key_fingerprint }}
+  shell: cat {{ tempdir.path }}/{{ key_fingerprint }} | gpg --dearmor > {{ tempdir.path }}/binary.gpg # noqa risky-shell-pipe
   when: key_needs_import
   changed_when: false
 
-- name: "Extract the required key from the binary keyring (when not {{ datadog_apt_key_current_name }})"
-  shell: "gpg --no-default-keyring --keyring {{ tempdir.path }}/binary.gpg --export {{ key_fingerprint }} > {{ tempdir.path }}/single.gpg"
+- name: Extract the required key from the binary keyring (when not {{ datadog_apt_key_current_name }})
+  shell: gpg --no-default-keyring --keyring {{ tempdir.path }}/binary.gpg --export {{ key_fingerprint }} > {{ tempdir.path }}/single.gpg
   when: key_fingerprint != datadog_apt_key_current_name and key_needs_import
   changed_when: false
 
-- name: "Extract the required key from the binary keyring (when {{ datadog_apt_key_current_name }})"
+- name: Extract the required key from the binary keyring (when {{ datadog_apt_key_current_name }})
   copy:
     src: "{{ tempdir.path }}/binary.gpg"
     dest: "{{ tempdir.path }}/single.gpg"
     mode: "0600"
-    remote_src: yes
+    remote_src: true
   when: key_fingerprint == datadog_apt_key_current_name and key_needs_import
   changed_when: false
 
-- name: "Import key {{ key_fingerprint }} to {{ datadog_apt_usr_share_keyring }} keyring"
-  shell: "cat {{ tempdir.path }}/single.gpg | gpg --no-default-keyring --keyring {{ datadog_apt_usr_share_keyring }} --import --batch"  # noqa risky-shell-pipe
+- name: Import key to keyring, key,keyring={{ key_fingerprint, datadog_apt_usr_share_keyring }}
```

Comment:
Yes indeed, ansible-lint wants all the variable in the name of the task to be put at the end. Because it is easier to search for the task name when all the variables are at the end.

https://ansible-lint.readthedocs.io/rules/name/

Reply:
Interesting, thanks for the pointer.